query/graphql/src/main/java/org/codice/ddf/admin/graphql/transform/GraphQLTransformOutput.java

```diff
@@ -200,4 +203,39 @@ public String createOutputObjectFieldTypeName(String fieldTypeName) {
                 transformEnum.getEnumTypeProvider(),
                 outputTypeProvider);
     }
+
+    private String toPrettyString(FunctionField<DataType> funcField,
```

Comment:
I might be good to add something to this method to make it clear it is invoking some sort of filtering.